scripts/pre_commit_linter.py

```diff
@@ -984,16 +986,25 @@ def _check_docstrings(all_files):
                 if line_num > 0:
                     prev_line = file_content[line_num - 1].lstrip().rstrip()
 
+                #Check for single line docstring starting with a space
+
+                if line.startswith(' """'):
```

Comment:
Shouldn't the condition be `if line.startswith('""" '):` ? We need to have a check so that there is not space _after_ the starting triple quotes of the docstring.